docs/source/model_design/general_research.rst

```diff
@@ -888,9 +888,125 @@ respiratory disease.
 Cross-Sectional Studies
 +++++++++++++++++++++++
 
-.. todo::
+Introduction
+~~~~~~~~~~~~
 
-	This section
+Cross-sectional studies are observational design in nature and give a snapshot 
+of the characteristics of study subjects at one point of time. In general, the 
+information can be gathered with respect to exposure, outcome alone, or both 
+concurrently. Unlike cohort studies, cross-sectional studies do not have a 
+follow-up period and therefore are relatively simple to conduct. As the exposure 
+status and outcome of interest information is collected in a single timepoint 
+often by surveys, it's quite difficult to determine the association between cause 
+and effect. We should not make a causal inference unless the exposure assumed to 
+be steady over time and not influenced by the outcome. This study design is 
+commonly used to assess the prevalence of a disease in a population, so called 
+`prevalence` study. Prevalence measures the prorpotion of population who have 
+the exposure or outcome of interest, it's calculated as the number of people 
+with the outcome in a population at specific point in time, divided by the 
+total number of people in that population at same time.
+
+
+Purpose of study
+~~~~~~~~~~~~~~~~
+
+Cross-sectional studies are used when we aim to find the prevalence of the 
+outcome of interest in a population or subgroups within the population at 
+specific timepoint; or describe a population or subgroups within the population 
+with respect to an outcomes and a set of risk factors. To investigate the 
+relationship between risk factors and the outcome of interest, research often 
+conducted in the form of a survey. Since the study performed at single moment of 
+time point, the limitation is obvious. It cannot show the sequence of events and 
+lack of information for a exposure in the past or future, therefore impossible 
+to infer causality. However, the cross-sectional studies are still useful in 
+designing hypotheses for future research.
+
+
+Types of study
+~~~~~~~~~~~~~~
+
+Cross-sectional studies can be either **descriptive** or **analytical**. 
+Descriptive studies evaluate the outcome or exposure alone, which collects the 
+information on frequency or distribution of health-related variables in a defined 
+population at one time point. (e.g. prevalence of cervical cancer among Chinese 
+women aged 40 to 45) It's a **point prevalence** measure if the data is collected 
+by asking a question "Do you have cervical cancer today?". In contrast, it would 
+become **period prevalence** if question changed to "Have you had cervical cancer 
```

Comment:
If think there's a second way to end up with a period prevalence----say the study asks 'do you have cervical cancer today' but they collect responses from visitors to a clinic over the course of a month.

It seems like in this scenario, the ratio of [the period of time during which you are collecting answers] to [the average duration of the condition or diseases] would define if you'd refer to this more as a point or period prevalence

also I don't have specific examples on hand of such studies (maybe they're quite rare, or just considered bad quality? though i can pull up some old lit searches and skim some methods sections if this is a valid point to dig into) but i do recall that when we read through @Ninicorn 's resources on point prev vs. period prev. vs incidence, sometimes it would refer to 'december 2019' as a 'point'.